src/main/kotlin/org/opensearch/replication/util/ValidationUtil.kt

```diff
@@ -161,4 +161,7 @@ object ValidationUtil {
         return clusterService.settings.getByPrefix(Node.NODE_ATTRIBUTES.key + RemoteStoreNodeAttribute.REMOTE_STORE_NODE_ATTRIBUTE_KEY_PREFIX).isEmpty == false
     }
 
+    fun isRemoteMigrating(clusterService: ClusterService): Boolean {
```

Comment:
Nit: `IsMigratingToRemote`

Reply:
This might be depicting a uni-directional migration which is DocRep to Remote. But migration direction can be both to Remote or to DocRep.

Reply:
ack